cranelift/interpreter/src/interpreter.rs

```diff
@@ -150,8 +150,13 @@ impl Interpreter {
                 let arg1 = frame.get(&args[0]);
                 let arg2 = frame.get(&args[1]);
                 let result = match opcode {
-                    Iadd => binary_op!(Add::add[arg1, arg2]; [I8, I16, I32, I64, F32, F64]; inst),
-                    Isub => binary_op!(Sub::sub[arg1, arg2]; [I8, I16, I32, I64, F32, F64]; inst),
+                    Iadd => binary_op!(Add::add[arg1, arg2]; [I8, I16, I32, I64]; inst),
```

Comment:
`iadd` supports `i128` as type. `isub` and `imul` don't have a legalization for `i128` yet, but it would be nice if they supported `i128` in the interpreter anyway.